packages/airnode-utilities/src/caching/index.ts

```diff
@@ -0,0 +1,207 @@
+import { existsSync, mkdirSync, readdirSync, readFileSync, rmdirSync, rmSync, statSync, writeFileSync } from 'fs';
+import { join } from 'path';
+import { logger } from '../logging';
+
+export const CACHE_BASE_PATH = `/tmp/airnode-cache`;
+
+/**
+ * Lambda, when last tested circa November 2021, persists /tmp contents for 2.5 hours.
+ */
+export const CACHE_MAX_FILESYSTEM_AGE_MINUTES = 60;
+
+/**
+ * This is the maximum number of files allowed in the cache.
+ * Lambda's /tmp mountpoint can handle approximately 130k files of minimum size (4KB).
+ * 4 KB is a common sector size for modern filesystems and is used on Lambda's /tmp mountpoint.
+ *
+ * 512 MB (size of Lambda /tmp) * 1024 / 4 KB (minimum size) = ~130k files
+ *
+ * Some URLs detailing the 4 KB minimum file size:
+ * ext4: https://askubuntu.com/questions/186813/why-does-every-directory-have-a-size-4096-bytes-4-k
+ * tmpfs: https://www.linuxquestions.org/questions/linux-server-73/tmpfs-block-size-4175501796/
+ * ntfs: https://support.microsoft.com/en-us/topic/default-cluster-size-for-ntfs-fat-and-exfat-9772e6f1-e31a-00d7-e18f-73169155af95#:~:text=By%20default%2C%20the%20maximum%20cluster,have%20a%20larger%20cluster%20size.
+ */
+export const CACHE_MAX_FILES = 20_000;
+
+export const isJest = () => process.env.JEST_WORKER_ID !== undefined;
+
+/**
+ * Initialise the path for the fs cache
+ */
+const initPath = () => {
+  try {
+    if (!existsSync(CACHE_BASE_PATH)) {
+      mkdirSync(CACHE_BASE_PATH);
+    }
+  } catch (e) {
+    logger.error(`Unable to init fs cache`);
+    logger.error((e as Error).stack!);
+  }
+};
+
+/**
+ * Get all keys from the cache that start with startingKey.
+ *
+ * @param prefix should take the form of `blockedRequestId-` and the full key could be `blockedRequestId-0x0000000...`
+ */
+const getKeys = (prefix?: string): string[] => {
+  try {
+    const keys = readdirSync(CACHE_BASE_PATH);
+    if (prefix) {
+      return keys.filter((key) => key.indexOf(prefix) === 0);
```

Comment:
Why not `.startsWith`?

Reply:
uh, because I don't know TS/JS that well ;)
My use of `indexOf` comes from Java bad habits :D 
I'll swap it out shortly.